### ISSUE TITLE
Add aria-hidden to <a> generated by `<CSVLink>` component

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5123]
## Area or Component
Results > (Click "Download results" button) > Download test results modal

## Quick Summary
An `<a>` generated by the `CSVLink` component with a `hidden` class has no discernible text.

## Accessibility Violation
[Links must have discernible text](https://dequeuniversity.com/rules/axe/4.6/link-name?application=AxeChrome)

## Action Requested
Add `aria-hidden="true"` to the component so it is hidden from screenreaders

## Screenshots (optional)
![Screen Shot 2023-01-25 at 16.12.32.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/29e77965-d466-4cf0-b985-5dc2f95a0d6d)

## Source Code (optional)

[which file or line is the issue attached to?]: #

## Original Deque Issue
[Links must have discernible text](https://dequeuniversity.com/rules/axe/4.6/link-name?application=AxeChrome)
